src/tags/object/RichText/view.js

```diff
@@ -363,15 +363,19 @@ class RichTextPieceView extends Component {
 
     // fix unselectable links
     const style = doc.createElement("style");
-
-    style.textContent = "body a { pointer-events: all; }";
+    const script = doc.createElement("script");
+
+    style.textContent = "body a { pointer-events: all !important; } .htx-highlight {display: inline !important;}";
+    script.textContent = `
+    document.querySelectorAll("a").forEach(a => a.removeAttribute("href"));
```

Comment:
Removing `href` is destructive, especially when there is `clickableLinks=true` param.
Possible solution: move `href` to `data-href` and when user clicks on it we can open new tab programmatically. Also we can add `title` with link url to show it to user.